### PR TITLE
Fix error messages for unmanaged attributes

### LINKED ISSUE #25977: No error message displayed when trying to add read-only attribute to some user in `Attributes` tab
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

- Enable user profile
- Enable switch `Unmanaged attributes` to `Enabled` for some realm
- Create some user. He has enabled `Attributes` tab in the admin console, which is correct
- Try to add attribute `LDAP_ID` and click `Save`. The server returns 400, which is correct behaviour. However UI does not display any error message. Would it be better if UI display some better error message for this case?

Just a note that I don't need LDAP provider created in my realm and I don't need attribute `LDAP_ID` to be defined on user profile. The attribute `LDAP_ID` is one of the attributes specified as read-only attribute in the user profile provider configuration https://www.keycloak.org/docs/latest/server_admin/index.html#read_only_user_attributes .

### Version

nightly

### Expected behavior

UI display some error message when the 400 is returned from the server when clicking `Save` on the `Attributes` page of some user

### Actual behavior

Nothing is displayed to the administrator, so he doesn't know if attribute was successfully added to the user etc (unless he monitors server log where is WARN message displayed).

### How to Reproduce?

See above

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by hmlnarik**
Fixes: #25977

This fixes multiple issues:
1. The `updateReadOnlyAttributesRejectedMessage` key is added to the correct bundle (first commit)
2. The form can be saved multiple times. Originally it was not possible to resubmit the form if errors were present on attributes and the form had to be reloaded
3. The errors are attached to the respective unmanaged attributes (see the image below)

image

### CODE REVIEW

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/components/key-value-form/KeyValueInput.tsx [hunk 1]
```diff
@@ -74,7 +74,11 @@ export const KeyValueInput = ({
         </GridItem>
         {fields.map((attribute, index) => {
           const keyError = !!(errors as any)[name]?.[index]?.key;
-          const valueError = !!(errors as any)[name]?.[index]?.value;
+          const valueErrorPresent =
+            !!(errors as any)[name]?.[index]?.value ||
+            !!(errors as any)[name]?.[index]?.message;
+          const valueError =
+            (errors as any)[name]?.[index]?.message || t("valueError");
```
[the remark this answers is not in the document]
Comment (hmlnarik) [reply js/apps/admin-ui/src/components/key-value-form/KeyValueInput.tsx:81]: Makes perfect sense

File: js/apps/admin-ui/src/user/EditUser.tsx [hunk 2]
```diff
@@ -147,9 +156,37 @@ export default function EditUser() {
       refresh();
     } catch (error) {
       if (isUserProfileError(error)) {
-        setUserProfileServerError(error, form.setError, ((key, param) =>
-          t(key as string, param as any)) as TFunction);
-        addError("userNotSaved", error);
+        const unmanagedAttributesInputs = form.getValues()[
```
[the remark this answers is not in the document]
Comment (hmlnarik) [reply js/apps/admin-ui/src/user/EditUser.tsx:159]: That would be good. My concern is re type of `data.unmanagedAttributes` which is `KeyValueType[] | Record<string, string | string[]>` while here I need to have `KeyValueType[]`. Can `data.unmanagedAttributes` be retyped just to `KeyValueType[]`?

```suggestion
const unmanagedAttributesInputs = data.unmanagedAttributes as KeyValueType[];
```

[the remark this answers is not in the document]
Comment (jonkoops) [reply js/apps/admin-ui/src/user/EditUser.tsx:159]: Yeah, that is an option. You could also narrow the type using a run-time check, which is the most 'safe'. For example:

```ts
const unmanagedAttributesInputs = data.unmanagedAttributes;

if (!Array.isArray(unmanagedAttributesInputs)) {
  return;
}

// TypeScript knows that unmanagedAttributesInputs is KeyValueType[]
```

[the remark this answers is not in the document]
Comment (hmlnarik) [reply js/apps/admin-ui/src/user/EditUser.tsx]: I fixed this along this suggestion and simplified a bit further, thanks for the hint!

**Review by jonkoops — APPROVED** [review]
LGTM

**Review by vmuzikar — APPROVED** [review]
Approving based on @jonkoops' review.

**Pull request merged**